### PR TITLE
lightning: merge lightning trouble shooting into faq

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
- Merge lighting page `troubleshoot` into `faq`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [x] Delete files
- [x] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
@3pointer @kennytm @overvenus PTAL

**Comment by glorv** [comment]
Some items are outdated, I will update them update this pr for easier review.

**Review by kennytm — COMMENTED** [review]
lgtm

File: tidb-lightning/tidb-lightning-faq.md [hunk 1]
```diff
@@ -1,16 +1,12 @@
 ---
 title: TiDB Lightning 常见问题
-aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/']
+aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/', '/docs-cn/dev/troubleshoot-tidb-lightning/','/docs-cn/dev/how-to/troubleshoot/tidb-lightning/','/docs-cn/dev/reference/tools/error-case-handling/lightning-misuse-handling/','/docs-cn/dev/tidb-lightning/tidb-lightning-misuse-handling/']
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:3]:
```suggestion
aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/''/docs-cn/dev/troubleshoot-tidb-lightning/','/docs-cn/dev/how-to/troubleshoot/tidb-lightning/','/docs-cn/dev/reference/tools/error-case-handling/lightning-misuse-handling/','/docs-cn/dev/tidb-lightning/tidb-lightning-misuse-handling/']
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 2]
```diff
@@ -61,7 +57,7 @@ Local-backend 和 Importer-backend 无需以上两个权限，因为数据直接
 4. 重启 `tikv-importer`。
 5. 重启 `tidb-lightning` 并等待，**直到程序因校验和错误（如果有的话）而失败**。
     * 重启 `tikv-importer` 将清除所有仍在写入的引擎文件，但是 `tidb-lightning` 并不会感知到该操作。从 v3.0 开始，最简单的方法是让 `tidb-lightning` 继续，然后再重试。
-6. [清除失败的表及断点](/troubleshoot-tidb-lightning.md#checkpoint-for--has-invalid-status错误码)。
+6. [清除失败的表及断点](/tidb-lightning/tidb-lightning-faq.md#checkpoint-for--has-invalid-status错误码)。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:60]:
```suggestion
6. [清除失败的表及断点](#checkpoint-for--has-invalid-status错误码)。
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 3]
```diff
@@ -191,10 +187,156 @@ upload-speed-limit = "100MB"
 
     如果出于某些原因而无法运行该命令，你可以尝试手动删除 `/tmp/tidb_lightning_checkpoint.pb` 文件。
 
-2. 删除 `tikv-importer` 所在机器上的整个 “import” 文件目录。
+2. 如果使用 Local Backend, 删除配置中 `sorted-kv-dir` 对应的目录; 如果使用 Importer Backend, 删除 `tikv-importer` 所在机器上的整个 “import” 文件目录。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:190]:
```suggestion
2. 如果使用 Local-backend，删除配置中 `sorted-kv-dir` 对应的目录；如果使用 Importer-backend，删除 `tikv-importer` 所在机器上的整个 `import` 文件目录。
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 4]
```diff
@@ -191,10 +187,156 @@ upload-speed-limit = "100MB"
...
 
 3. 如果需要的话，删除 TiDB 集群上创建的所有表和库。
 
 ## TiDB Lightning 报错 `could not find first pair, this shouldn't happen`
 
 报错原因是遍历本地排序的文件时出现异常，可能在 lightning 打开的文件数量超过系统的上限时发生。在 linux 系统中，可以使用 `ulimit -n` 命令确认此值是否过小。建议在 lightning 导入期间将此设置调整为 1000000（`ulimit -n 1000000`）。
+
+## Lightning 导入速度太慢
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:198]:
```suggestion
## TiDB Lightning 导入速度太慢
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 5]
```diff
@@ -1,16 +1,12 @@
 ---
 title: TiDB Lightning 常见问题
-aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/']
+aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/', '/docs-cn/dev/troubleshoot-tidb-lightning/','/docs-cn/dev/how-to/troubleshoot/tidb-lightning/','/docs-cn/dev/reference/tools/error-case-handling/lightning-misuse-handling/','/docs-cn/dev/tidb-lightning/tidb-lightning-misuse-handling/','/zh/tidb/dev/tidb-lightning-faq/']
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:3]:
```suggestion
aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/','/docs-cn/dev/troubleshoot-tidb-lightning/','/docs-cn/dev/how-to/troubleshoot/tidb-lightning/','/docs-cn/dev/reference/tools/error-case-handling/lightning-misuse-handling/','/docs-cn/dev/tidb-lightning/tidb-lightning-misuse-handling/','/zh/tidb/dev/tidb-lightning-faq/']
```

File: tidb-lightning/tidb-lightning-faq.md [hunk 6]
```diff
@@ -191,10 +187,156 @@ upload-speed-limit = "100MB"
 
     如果出于某些原因而无法运行该命令，你可以尝试手动删除 `/tmp/tidb_lightning_checkpoint.pb` 文件。
 
-2. 删除 `tikv-importer` 所在机器上的整个 “import” 文件目录。
+2. 如果使用 Local-backend, 删除配置中 `sorted-kv-dir` 对应的目录; 如果使用 Importer-backend, 删除 `tikv-importer` 所在机器上的整个 `import` 文件目录。
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:190]:
```suggestion
2. 如果使用 Local-backend，删除配置中 `sorted-kv-dir` 对应的目录；如果使用 Importer-backend，删除 `tikv-importer` 所在机器上的整个 `import` 文件目录。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: tidb-lightning/tidb-lightning-faq.md [hunk 7]
```diff
@@ -1,16 +1,12 @@
 ---
 title: TiDB Lightning 常见问题
-aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/']
+aliases: ['/docs-cn/dev/tidb-lightning/tidb-lightning-faq/','/docs-cn/dev/faq/tidb-lightning/','/docs-cn/dev/troubleshoot-tidb-lightning/','/docs-cn/dev/how-to/troubleshoot/tidb-lightning/','/docs-cn/dev/reference/tools/error-case-handling/lightning-misuse-handling/','/docs-cn/dev/tidb-lightning/tidb-lightning-misuse-handling/','/zh/tidb/dev/tidb-lightning-faq/']
```
Comment (TomShawn) [inline tidb-lightning/tidb-lightning-faq.md:3]: broken link fixed in https://github.com/pingcap/docs-cn/pull/5345.

**Pull request merged**